### DISCUSSION TITLE
add support in Blinka for orange pi zero 3

### DISCUSSION THREAD

**Discussion by skyfsj** [adafruit/Adafruit_Blinka#1089] [New Board Requests]
this gpio pinout
![image](https://github.com/adafruit/Adafruit_Blinka/assets/47785500/ddb43586-3364-4243-87e4-05c6e67bcb72)

board doc: https://drive.google.com/drive/folders/1pZ4X4uP2C7N_6wXU0TjAxC7U1eCG05L1

**Comment by thuvasooriya** [comment]
I guess I can second this here since the Orangepi Zero 2w also have more or less the same system specs except the pinout, 
I want support for Orangepi Zero 2w, I'll add a separate issue later to track it too.
if anyone can suggest me how to get started on adding support for a new board I would like to work on this and make a contribution

**Comment by sp2ong** [comment]
Hi

We try use connected OLED 0.96 via I2C to Orange Pi Zero 3 and it looks like board is not detected via PlatformDetect

     File "/opt/fmpoland/oled/test.py", line 13, in <module>
        import board
      File "/usr/local/lib/python3.11/dist-packages/board.py", line 405, in <module>
          raise NotImplementedError(
        NotImplementedError:
        Adafruit-PlatformDetect version 3.62.0 was unable to identify the board and/or
        microcontroller running the Linux platform. Please be sure you
        have the latest packages by running:
        'pip3 install --upgrade adafruit-blinka adafruit-platformdetect
        If you are running the latest package, your board may not yet be supported. Please
        open a New Issue on GitHub at https://github.com/adafruit/Adafruit_Blinka/issues and
        select New Board Request.

When I run example PlatformDetect code:

      from adafruit_platformdetect import Detector
      detector = Detector()
      print("Chip id: ", detector.chip.id)
      print("Board id: ", detector.board.id)

Result show only

       Chip id: H6

no bard info is displayed

It's nice when support for the Orange Pi Zero 3 computer is added to be able to use the adafruit library to display information on OLED

Regards

Waldek

**Comment by nopnop2002** [comment]
@sp2ong 

>if anyone can suggest me how to get started on adding support for a new board I would like to work on this and make a contribution

Adding a Single Board Computer to PlatformDetect for Blinka
https://learn.adafruit.com/adding-a-single-board-computer-to-platformdetect-for-blinka/overview
